src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/letter/AppellantInternalLateRemissionPartiallyOrGrantedLetterPersonalisationTest.java

```diff
@@ -109,7 +112,10 @@ public void setup() {
 
     @Test
     void should_return_given_template_id() {
-        assertEquals(letterTemplateId, appellantInternalLateRemissionPartiallyOrGrantedLetterPersonalisation.getTemplateId());
+        if (Objects.equals(asylumCase.read(REMISSION_DECISION, RemissionDecision.class), Optional.of(RemissionDecision.APPROVED)))
+            assertEquals(approvedLetterTemplateId, appellantInternalLateRemissionPartiallyOrGrantedLetterPersonalisation.getTemplateId(asylumCase));
+        else if (Objects.equals(asylumCase.read(REMISSION_DECISION, RemissionDecision.class), Optional.of(RemissionDecision.PARTIALLY_APPROVED)))
+            assertEquals(partiallyApprovedLetterTemplateId, appellantInternalLateRemissionPartiallyOrGrantedLetterPersonalisation.getTemplateId());
```

Comment:
Just for code coverage, would it be an idea to test the Rejected use case here and also error when the REMISSION_DECISION is null use case?